google-cloud-dlp/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+### 0.6.3 / 2018-09-28
+
+* Update google-cloud-dlp generated files (#2481)
+* Re-generate library using google-cloud-dlp/synth.py (#2478)
```

Comment:
We need to do a better job at updating the merge commit to make a meaningful description of the changes. IMO.